src/app/shared/components/template/components/round-icon-button/round-icon-button.component.scss

```diff
@@ -1,12 +1,16 @@
 @use "/src/theme/mixins";
 
+$round-icon-button-light: var(--round-icon-button-light, var(--ion-color-yellow));
```

Comment:
Here I'd probably recommend the default always being one of primary or secondary colours (probably best to stick mostly with shades of primary) and make the override at the theme level to the yellow colour (which I think is sensible with the way you have added and renamed).